src/context.h

```diff
@@ -43,7 +43,7 @@ namespace librealsense
     class device_info
     {
     public:
-        virtual std::shared_ptr<device_interface> create_device(bool register_device_notifications = false) const
+        virtual std::shared_ptr<device_interface> create_device(bool register_device_notifications = true) const
```

Comment:
Have we checked that we always use the default arg?

Reply:
This parameter was removed, create device will always create it with a callback.
Verified playback work as expected.

Reply:
network device does not inherent device so it's good too.